tests/test_flow_studio_ui_ids.py

```
@@ -87,7 +87,7 @@ def extract_uiids_from_html(html: str) -> List[Tuple[str, int]]:
 
     # Track whether we're inside a script tag
     in_script = False
-    script_start = re.compile(r"<script\b", re.IGNORECASE)
+    script_start = re.compile(r"<script(?!.*data-inline-source)", re.IGNORECASE)
```

Ruff (0.15.9)

[error] Invalid unescaped character backspace, use "\b" instead

(PLE2510)